systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthPlainIsolatedST.java

```diff
@@ -727,12 +763,24 @@ void setUp(ExtensionContext extensionContext) {
 
         keycloakInstance.setRealm("internal", false);
 
+        // This was done due to different pattern used for oauth metrics
+        ConfigMap kafkaMetricsCm = TestUtils.configMapFromYaml(TestUtils.USER_PATH + "/src/test/resources/oauth2/oauth-metrics.yaml", "kafka-metrics");
```

Comment:
hmm, you may guess what's missing :D
```suggestion
       final  ConfigMap kafkaMetricsCm = TestUtils.configMapFromYaml(TestUtils.USER_PATH + "/src/test/resources/oauth2/oauth-metrics.yaml", "kafka-metrics");
```